### PR TITLE
Cleaning up comments and unused arguments in setup.py

### PR DESCRIPTION
**Pull request by NivekT**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #37

Cleaning up comments and unused arguments in `setup.py`. Removing custom `clean` class as it is not needed.

Differential Revision: [D31381731](https://our.internmc.facebook.com/intern/diff/D31381731)

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -90,12 +90,9 @@ def run(self):
         "Topic :: Scientific/Engineering :: Artificial Intelligence"
     ],
     # Package Info
-    packages=find_packages(exclude=["build*", "test*", "benchmark*", "third_party*", "build_tools*",
-                                    "datapipes_old", "examples", "scripts"]),
+    packages=find_packages(exclude=["build*", "test*", "benchmark*", "third_party*", "examples"]),
```
Comment (ejguan) [inline setup.py:93]: nit: We can also remove `benchmark`. Let's keep `third_party` for future development

File: setup.py [hunk 2]
```diff
@@ -90,12 +90,9 @@ def run(self):
...
     zip_safe=False,
-    # ext_modules = ???  # TODO: do we need external modules?
     cmdclass={
-        # 'build_ext': ???  # TODO: if we need external modules
         'clean': clean,
```
Comment (ejguan) [inline setup.py:96]: We don't have `build` and extension for now. Could we just use default clean?

**Comment by NivekT** [comment]
@nivekt has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D31381731).

**Comment by NivekT** [comment]
@nivekt has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D31381731).

**Comment by NivekT** [comment]
@nivekt has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D31381731).

**Pull request closed without merging**